articles/app-service-mobile/app-service-mobile-xamarin-forms-get-started-users.md

```diff
@@ -219,14 +246,25 @@ This section shows how to implement the **IAuthenticate** interface in the iOS a
         }
 
     If you are using an identity provider other than Facebook, choose a different value for [MobileServiceAuthenticationProvider].
+
+6. Add OpenUrl(UIApplication app, NSUrl url, NSDictionary options) method overload
```

Comment:
Change wording to "Update the **AppDelegate** class by adding OpenUrl(UIApplication app, NSUrl url, NSDictionary options) method overload"